app/frontend/components/module_assistant/step_update_repositories.js

```diff
@@ -0,0 +1,186 @@
+/* This file is part of Ezra Bible App.
+
+   Copyright (C) 2019 - 2021 Ezra Bible App Development Team <contact@ezrabibleapp.net>
+
+   Ezra Bible App is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 2 of the License, or
+   (at your option) any later version.
+
+   Ezra Bible App is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with Ezra Bible App. See the file LICENSE.
+   If not, see <http://www.gnu.org/licenses/>. */
+
+
+const { html } = require('../../helpers/ezra_helper.js');
+const assistantController = require('./assistant_controller.js');
+const i18nController = require('../../controllers/i18n_controller.js');
+const assistantHelper = require('./assistant_helper.js');
+
+const template = html`
+<style>
+  .update-repository-data-wrapper {
+    padding: 1em 2.5%;
+    min-height: 6em;
+    background: var(--widget-bg-color, #eee);
+    border-radius: 5px;
+  }
+  #update-repository-data-info {
+    text-align: center;
+  }
+  #update-repository-data-info .intro{
+    margin: 0 0 1em;
+  }
+
+  #update-repository-data-progress p, #update-repository-data-progress .progress-bar{
+    margin: 0;
+  }
+</style>
+
+<section class="update-repository-data-wrapper">
+  
+  <div id="update-repository-data-info" class="info-view">
+    <p class="intro"></p>   
+    <span class="update-info"></span>
+    <button id="update-repo-data" class="fg-button ui-state-default ui-corner-all" i18n="module-assistant.update-now"></button>
+  </div>
+  
+  <div id="update-repository-data-progress" class="update-view"> 
+    <p i18n="module-assistant.updating-repository-data"></p>
+    <div id="repo-update-progress-bar" class="progress-bar">
+      <div class="progress-label" i18n="module-assistant.updating"></div>
+    </div>
+  </div>  
+  <p id="update-failed" style="display: none" i18n="module-assistant.update-repository-data-failed"></p>
+
+</section>
+`;
+
+class StepUpdateRepositories extends HTMLElement {
```

Comment:
This is technically not a "step" in the workflow like the others? Maybe we should remove the term "Step" from this component?